CHANGELOG.md

```diff
@@ -26,9 +27,11 @@ Special thanks to: @rejas, @sdetweil
 - Wait for all modules to start before declaring the system ready (#2487)
 - Updated e2e tests (moved `done()` in helper functions) and use es6 syntax in all tests
 - Updated da translation
+- The `cors`-method in `server.js` now supports sending and recieving HTTP headers.
```

Comment:
could we split up the yr provider from the rest of the changes? so we can take a better look at each before merging?
havin two or more features in a PR makes things sometimes easy to miss

Reply:
Sure 👍  Looking at the code coverage it looks like I have to learn Jest as well. I'll put them up as separate PRs, but probably not until next week 😊

Reply:
Added the support for HTTP-headers in https://github.com/MichMich/MagicMirror/pull/2957. Will change this PR to reflect the changes.